README.md

```diff
@@ -63,3 +63,72 @@ And poof, you are ready to make a PR with the updates.
 ## Test environments
 
 Template projects are hooked up to Platform.sh projects, so each new PR gets built as a new, empty environment ready for testing.  In most cases simply visiting the built site and verifying that the installer can run (if available) or that the site gives the appropriate "there's nothing here yet" error is sufficient, but further testing can be done if needed.
+
+
+## Contributing
+
+If you would like to contribute to the list of Platform.sh's maintained templates, there is a simple process for ensuring that future maintenance is set up from the start through `template-builder`.
+
+Let's use the previous example: you have created a new application that uses the framework Spiffy that you think would be a useful template. 
+
+1. Create a repository under the [Platform.sh Github organization](https://github.com/platformsh/) for the template. Officially maintained templates begin with the `template-` prefix, so name it `template-spiffy`, and initialize it with an empty `README.md`.
+2. Add the "Examples and Templates team" as `Admin` collaborators in the `platformsh/template-spiffy` repository settings.
```

Comment:
I think I usually use Write, not Admin.  But it probably should be Admin, actually.

Reply:
I think I started using `Admin` because I saw it on another repo, but maybe I'm wrong. Let's set in stone what we want though.